### DISCUSSION TITLE
[Laughs], [sigh], what else work for nudging Bark to produce interesting sound effects?

### DISCUSSION THREAD

**Discussion by ksylvan** [suno-ai/bark#265] [General]
What other non-verbal cues/clues are supported? I know at least [laughs], and [sighs] works but what other words are supported in brackets invoke different sounds?

The README lists these:

- [laughter]
- [laughs]
- [sighs]
- [music]
- [gasps]
- [clears throat]
- — or ... for hesitations
- ♪ for song lyrics
- CAPITALIZATION for emphasis of a word
- [MAN] and [WOMAN] to bias Bark toward male and female speakers, respectively

In the discussion of https://github.com/suno-ai/bark/issues/190 is a list (produced by ChatGPT):

```
[whispers], [giggles], [snickers], [coughs], [groans], [yells], [gasps], [whimpers], [sobs], [murmurs],
[chuckles], [hums], [sneezes], [grunts], [shrieks], [hiccups], [stammers], [stutters], [grumbles], [snorts],
[howls], [moans], [guffaws], [sighs deeply], [laughs nervously], [cries], [sniffs], [smacks lips], [claps], [sings],
[yawns], [mumbles], [shushes], [exhales sharply], [snaps], [whistles], [crunches], [slurps], [smacks], [clicks], 
[clinks], [clatters], [sizzles], [rustles], [splashes], [claps], [taps], [thumps], [rumbles], [drums], [jingles], [jangles],
[pops], [bangs], [hisses], [scratches], [squeaks], [screeches], [buzzes], [swooshes], [swoops], [clangs],
[whirrs], [chirps], [beeps], [tick-tocks], [thuds], [swishes], [crackles], [fizzes], [humming].
```

And then we have other hints that sometimes work, like `[upbeat music]`.

have you tried any of these and had any luck?

**Comment by panyanyany** [comment]
you can find more here [List of Metatags | SunoAI Wiki](https://sunoaiwiki.com/en/resources/2024-05-13-list-of-metatags/)
